github/GitRelease.py

```diff
@@ -170,6 +170,16 @@ def zipball_url(self):
         self._completeIfNotSet(self._zipball_url)
         return self._zipball_url.value
 
+    @property
+    def assets(self):
+        """
+        Already returned assets info (no additional requests will be done).
```

Comment:
Just the :type: here, please

Reply:
Where to put a note on the difference between `get_assets()` and `assets` though?